test/e2e/specs/editor/various/inserting-blocks.spec.js

```diff
@@ -388,4 +445,23 @@ class InsertingBlocksUtils {
 			'data-testid=block-draggable-chip >> visible=true'
 		);
 	}
+	async dragOver( boundingBox ) {
+		// Call the move function twice to make sure the `dragOver` event is sent.
+		// @see https://github.com/microsoft/playwright/issues/17153
+		for ( let i = 0; i < 2; i += 1 ) {
+			await this.page.mouse.move(
+				// Hover on the right side of the block to avoid collapsing with the preview.
+				boundingBox.x + boundingBox.width - 1,
+				// Hover on the bottom of the paragraph block.
+				boundingBox.y + boundingBox.height - 1
+			);
+		}
+	}
+
+	async expectIndicatorBelowParagraph( paragraphBoundingBox ) {
+		// Expect the indicator to be below the paragraph block.
+		await expect
+			.poll( () => this.indicator.boundingBox().then( ( { y } ) => y ) )
+			.toBeGreaterThan( paragraphBoundingBox.y );
+	}
```

Comment:
I have grouped similar operations performed in each test into the utility. This approach is also [used in the paragraph block](https://github.com/WordPress/gutenberg/blob/1783a4e15a0d0806151ed3b88fec8cded2886d5a/test/e2e/specs/editor/blocks/paragraph.spec.js#L521-L527).